data/conf/hive-log4j2.properties

```diff
@@ -14,9 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-status = INFO
```

Comment:
Why this changed?

Reply:
How do you test this change is ok? Do we have unit tests to verify this?

Reply:
- `status` is only used to set the output of log4j2's internal log. Its default value is `WARN`. Setting `status` to `INFO` actually masks the deprecated property warning of log4j2. When I looked at `mvn test -Dtest=TestHplsqlLocal`, I found that the assertion of the unit test is essentially matching the predefined output log file. In this case, there is actually no need to change the value of `status`. And starting from log4j2 2.24.0, the status attribute in the Configuration element has been deprecated, refer to https://logging.apache.org/log4j/2.x/manual/status-logger.html#config .
- The `log4j.debug` system property directly overrides changes to the `status` property.

Reply:
> How do you test this change is ok? Do we have unit tests to verify this?

- More discussion is at https://github.com/apache/logging-log4j2/issues/2794 . More discussion is at https://github.com/apache/logging-log4j2/issues/2794 . There is even a spinoff PR https://github.com/apache/hive/pull/5381 to improve unit testing.

Reply:
Can we replace the deprecated attribute  `status ` to `statusLoggerLevel`?

https://logging.apache.org/log4j/2.x/manual/status-logger.html#config
`Since version 2.24.0, status attribute in the Configuration element is deprecated and should be replaced with the log4j2.statusLoggerLevel configuration property.`

https://logging.apache.org/log4j/2.x/manual/status-logger.html#log4j2.statusLoggerLevel


It will be convinent for people to change the log status.

Reply:
@zhangbutao So, do you want to put `<log4j2.statusLoggerLevel>INFO</log4j2.statusLoggerLevel>` in the top-level `pom.xml` to apply globally? Otherwise, a separate `log4j2.StatusLogger.properties` file is required.

Reply:
https://github.com/apache/logging-log4j2/issues/2794#issuecomment-2267564572 

`<log4j.debug>true` line in `/pom.xml`  and the `status` should be changed at the same?

Can we only keep the `status`? (can replace with the new property)

Reply:
OK. i see. Thanks.